ibm-mq-mixin/dashboards/queue-manager-overview.libsonnet

```diff
@@ -0,0 +1,1271 @@
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      '(ibmmq_qmgr_ram_total_estimate_for_queue_manager_bytes{mq_cluster=~"$mq_cluster", qmgr=~"$qmgr", job=~"$job"}/ibmmq_qmgr_ram_total_bytes{mq_cluster=~"$mq_cluster", qmgr=~"$qmgr", job=~"$job"}) * 100',
+      datasource=promDatasource,
+      legendFormat='{{job}} - {{mq_cluster}}: {{qmgr}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Estimated memory utilization',
+  description: 'The estimated memory usage of the queue managers.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'percent',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local queueManagerStatusPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'ibmmq_qmgr_uptime{mq_cluster=~"$mq_cluster", qmgr=~"$qmgr", job=~"$job"}',
+      datasource=promDatasource,
+      format='table',
+    ),
+    prometheus.target(
+      'ibmmq_qmgr_status{mq_cluster=~"$mq_cluster", qmgr=~"$qmgr", job=~"$job"}',
+      datasource=promDatasource,
+      legendFormat='',
+    ),
+  ],
+  type: 'table',
+  title: 'Queue manager status',
+  description: 'A table showing the status and uptime of the queue manager.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      custom: {
+        align: 'auto',
+        cellOptions: {
+          type: 'auto',
+        },
+        filterable: false,
+        inspect: false,
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [
```

Comment:
just checking that the overrides in here are intentional

Reply:
yup, they were, had to include a unit into the table